pkg/generated/openapi/def.bzl

```diff
@@ -1,7 +1,7 @@
 load("@io_bazel_rules_go//go:def.bzl", "go_library")
 load("@io_kubernetes_build//defs:go.bzl", "go_genrule")
 
-def openapi_library(name, tags, srcs, openapi_targets=[], vendor_targets=[]):
+def openapi_library(name, tags, srcs, openapi_targets=[], vendor_targets=[], input_prefix="k8s.io/kubernetes/", output_prefix="", pkg_prefix=""):
```

Comment:
With these default values, do you even need the openapi.bzl file above? It seems like you're duplicating there what you've defaulted to here.

Reply:
I was following the python practice of making new parameters optional with sane defaults.  ``openapi.bzl`` is required to allow a vendoring repo to override the parameters.

Reply:
Should I make the parameters mandatory instead?

Reply:
I prefer mandatory parameters, but I'll defer to @ixdy if he has a strong opinion.

If this were a BUILD rule being used extensively elsewhere, then I'd say optional parameters would be a better approach, but these are not IMO conceptually-optional parameters and you can fix all callers easily (since there's only one).

Reply:
Done.